day_two_guide/topics/proc_backing-up-registry-certs.adoc

```diff
@@ -0,0 +1,37 @@
+////
+Backing up registry certificates
+
+Module included in the following assemblies:
+
+* day_two_guide/environment_backup.adoc
+////
+
+[id='backing-registry-certificates_{context}']
+= Backing up registry certificates
+
+If you use an
+xref:../install_config/registry/securing_and_exposing_registry.adoc#exposing-the-registry[external
+secured registry], you must save all the registry certificates. The registry is
+secured by default.
+
+[IMPORTANT]
+====
+You must perform the following steps on each cluster node.
+====
+
+. Back up the registry certificates:  
++
+----
+# cd /etc/docker/certs.d/
+# tar cf /tmp/docker-registry-certs-$(hostname).tar *
+----
+
+. Move the backup to an external location.
```

Comment:
s./: after location

Reply:
Since there's not an example after this step, I don't think this swap makes sense.